README.md

```diff
@@ -216,6 +216,36 @@ By default: `['GET', 'HEAD', 'OPTIONS', 'TRACE' ]`
 
 This plugin will always retry on 503 errors, _unless_ `retryMethods` does not contain `GET`.
 
+#### `globalAgent`
+
+Enables the possibility to explictly opt-in for global agents.
+
+Usage for undici global agent:
+```js
+import { setGlobalDispatcher, ProxyAgent } from 'undici'
+
+const proxyAgent = new ProxyAgent('my.proxy.server')
+setGlobalDispatcher(proxyAgent)
+
+fastify.register(FastifyReplyFrom, {
+  base: 'http://localhost:3001/',
+  globalAgent: true, 
+  undici: {
+  }
+})
+```
+
+Usage for http/https global agent:
+```js
+fastify.register(FastifyReplyFrom, {
+  base: 'http://localhost:3001/',
+  // http and https is allowed to use http.globalAgent or https.globalAgent
+  globalAgent: true, 
+  http: {
+  }
```

Comment:
why is this empty object needed? It's not user friendly.

Reply:
Actually, it should not be an empty object. Passing `http: <something>` is the proper usage for using `http.request`.

We can use `http: true` or similar, but I found it would open way too many combination of option.
https://github.com/fastify/fastify-reply-from/blob/403328a34ae4c5b81997bd25a958958618bf2721/README.md?plain=1#L116-L153